python/fatcat_import.py

```diff
@@ -59,7 +59,7 @@ def run_crossref(args: argparse.Namespace) -> None:
             args.kafka_hosts,
             args.kafka_env,
             "api-crossref",
-            "fatcat-{}-import-crossref".format(args.kafka_env),
+            f"fatcat-{args.kafka_env}-import-crossref",
```

Comment:
Wow! Having automation for these f-string cleanups is pretty cool. I was considering starting to do many of these by hand, but via automation is pretty nice.

One question/concern I have before doing this in bulk is whether the existing static analysis (lint) in this repository will catch undefined variable use within f-strings, and whether use of variables in an f-string counts against "unused variable" lints. I know tooling could/should detect this kind of use, but i'm not sure if it works today. As an example with this snippet, if `args.kafka_env` was refactored to `args.generic_env` or something like that elsewhere, would the static analysis help with catching all the usages in the f-strings?

I'm generally for these f-string conversions, and they seem to be the bulk of the PR.

Reply:
The f-strings are easier to read and they have an [impressive performance boost](https://www.scivision.dev/python-f-string-speed) over other methods.

https://pypi.org/project/flynt also does f-string conversions but I have not used it.

`flake8`, `pylance`, `pylint`, and `python` all flag `bb` as an undefined name.
```
a = "apple"
b = "ball"
c = "cow"
print(f"My strings are: {a}, {bb}, {c}")
```